### DISCUSSION TITLE
Amstrad PC1512 - How to get Backslash ?

### DISCUSSION THREAD

**Discussion by Neelix57** [86Box/86Box#1617] [Q&A] [answered]
Hi,

I emulate the Amstrad PC1512 on 86box under wine on Xubuntu 18.04. It works very well.But, I can't get the backslash key to work.
 displays '^U" instead.

Could someone help please ?

**Comment by jvernet** [comment, marked as answer]
I’m using ALT-092 on the numpad.
Never found it on the emulated keyboard

Envoyé de mon iPhone

Le 20 août 2021 à 08:12, Neelix57 ***@***.***> a écrit :

﻿

Hi,

I emulate the Amstrad PC1512 on 86box under wine on Xubuntu 18.04. It works very well.But, I can't get the backslash key to work.
displays '^U" instead.

Could someone help please ?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/86Box/86Box/discussions/1617>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AD4OSKE5PX7NHFE4LARKF2DT5XW6BANCNFSM5CPU3NOA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>.

**Comment by Neelix57** [comment]
Thank you very much jvernet, it works.
